maintainers/maintainer-list.nix

```diff
@@ -10641,6 +10641,12 @@
     githubId = 35086;
     name = "Jonathan Wright";
   };
+  quantenzitrone = {
+    email = "quantenzitrone@protonmail.com";
+    github = "Quantenzitrone";
```

Comment:
Please also add githubId.

Reply:
But it is stated as optional in the comment at the beginning of maintainers/maintainers-list.nix ... ?